getting-started-guides/python/Uninstrumented/readme.md

```diff
@@ -76,5 +77,19 @@ Set the Application Name and New Relic [Ingest - License Key](https://docs.newre
    ```bash
    ./load-generator.sh
    ```
-   
+
+5. This will only collect logs. In order to get traces and metrics you need to [make the following update to app.py](https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/flask/flask.html#id1)
+    ```
+    from opentelemetry.instrumentation.flask import FlaskInstrumentor
+
+    app = Flask(__name__)
+
+    FlaskInstrumentor().instrument_app(app)
+    ```
```

Comment:
@pnvnd @reese-lee Can you help confirm why (or why not) this code change is necessary?

My understanding is that these steps roughly align with the getting started guide you authored - specifically [Tutorial 2](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-python/#tutorial-2-monitor-the-demo-app-with-the-opentelemetry-python-agent-python-agent) which demonstrates using Python's auto-instrumentation. I thought the whole point of auto-instrumentation was that no code change should be required.

Reply:
I have also tried [Otel's own getting started auto-instrumentation example](https://opentelemetry.io/docs/instrumentation/python/getting-started/)... and in fact I _don't_ have to add this to their dice server to make the auto instrumentation work for metrics & traces. They are both very simple examples with the only real difference being that the NR example has

    app.run(host='0.0.0.0', port=8080)
    
 And suggests invocation via `python app.py` and the otel example does not contain the app.run line and uses `flask run`. However. Commenting out that line and invoking this example with `flask run` does not seem to change the situation here. So I instead tried running this example with the virualenv I created via the otel tutorial... and bingo. 
 
 The otel [tutorial tells you to use the bootstrap](https://opentelemetry.io/docs/instrumentation/python/getting-started/#instrumentation) command to get the correct dependencies... but apparently you need more than _just_ the flask auto-instrumentation here to get these examples to generate metrics and traces.
 
So we could update these instructions to copy that. Although with a nearly identical example in the offical otel docs... maybe it would be easier for us to maintain a link to the otel quickstart with a note about the env variables/command to use to get the data into NR

Reply:
Hello @boylea (and @alanwest!)! I apologize for the delay, I was out of the office for a while. It has been quite some time since I worked on this with Peter; please allow me a bit more time to go through this PR and see what I can find out for you. 

In regards to this:

> Also note that there is some overlapping instructions for this tutorial on the [New Relic docs site here](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-python/#tutorial-2-monitor-the-demo-app-with-the-opentelemetry-python-agent-python-agent), and it is probably worth maintaining a single copy of these instructions and providing a link in one way or another.

Thank you for this feedback! Our thinking was to enable users to be able to stay on our Docs site if that's how they found the tutorial, without having to pass them back and forth between our Docs site and Examples repo, and similarly if they find the tutorial in our Examples repo. We recognize that it's duplicatory, but it is intended to save the user the back-and-forth; that said, we're open to changing things as needed. We'll likely leave things as they are for now, but have taken your feedback into consideration.